get_involved.md

```diff
@@ -6,30 +6,48 @@ layout: default
 
 # How to get involved
 
-Our goal is to make every effort going on as transparent and open as possible. Our official communication channel is the hep-sf-forum. Simply send a mail with ‘subscribe’ as the subject (not content) to hep-sf-forum+subscribe@googlegroups.com
+Our goal is to make every effort going on as transparent and open as possible.
+Our official communication channel is the hep-sf-forum. Simply send a mail with
+‘subscribe’ as the subject (not content) to
+hep-sf-forum+subscribe@googlegroups.com
 
-In addition, there is a HEP S&C community mailing list for occasional HEP software and computing community mailings. Everyone involved or interested in HEP S&C is encouraged to sign up to this list. It will be used for occasional mailings of community wide interest, such as announcements of HEP S&C conferences, workshops and schools.
+In addition, there is a HEP S&C community mailing list for occasional HEP
+software and computing community mailings. Everyone involved or interested in
```

Comment:
I'd say: `...community mailings. It is not strictly related to HSF: everyone involved...`